### PR TITLE
chore: disable dst fix

### PR DESCRIPTION
**Pull request by diogogmatos**
Disabled the Daylight Savings Time bug fix, that subtracts 1 hour to events inside DST to account for the old Netlify timezone bug, as it seems to be now acting like normal, causing future events (in mid february) to start displaying their time incorrectly (1h before the actual time).

Also updated node to version 20.0.0 and plugin-nextjs to address this Netlify warning:

image

### CODE REVIEW

**Review by ruilopesm — APPROVED** [review]

File: .tool-versions [hunk 1]
```diff
@@ -1 +1 @@
-nodejs 18.12.0
+nodejs 20.0.0
```
Comment (ruilopesm) [inline .tool-versions:1]:
```suggestion
nodejs 20.9.0
```

File: netlify.toml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 [build.environment]
   TZ = "Europe/Lisbon"
-  NODE_VERSION = "18.12.0"
+  NODE_VERSION = "20.0.0"
```
Comment (ruilopesm) [inline netlify.toml:7]:
```suggestion
  NODE_VERSION = "20.9.0"
```

**Comment by ruilopesm** [comment]
Always use LTS versions.

**Comment by diogogmatos** [comment]
[chore: disable dst fix](https://trello.com/c/5Mv9AIAN/77-chore-disable-dst-fix)

**Pull request merged**